Dockerfile

```diff
@@ -15,16 +16,15 @@ RUN apt-get update && apt-get install -y \
     python-ldap \
     python-memcache \
     python-pysqlite2 \
+    python-pip \
     python-simplejson \
-    python-support \
     python-twisted \
     sqlite3 \
     supervisor \
     unzip \
     wget
 
-RUN easy_install pip
-RUN pip install --upgrade pip
+RUN pip install --upgrade pip==20.1
```

Comment:
according to https://pypi.org/project/pip/#history `20.1.1` would be available as latest patch release for that minor version... i assume we have no reason to use `20.1` specifically?
we might want to add a comment on why we restricted the max version here... i assume it has to do with python2 support?

Reply:
👍